examples/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[workspace]
+members = ["contracts/*"]
+
+[workspace.package]
+version       = "0.1.0"
+edition       = "2021"
+license       = "Apache-2.0"
+repository    = "https://github.com/Finschia/fnsa-contracts"
+homepage      = "https://cosmwasm.com"
+documentation = "https://docs.cosmwasm.com"
+
+[workspace.dependencies]
+cosmwasm-schema  = "1.1.9"
+cosmwasm-std     = "1.1.9"
+cosmwasm-storage = "1.1.9"
+cw-storage-plus  = "1.1.0"
+cw2              = "1.1.2"
+finschia-std     = { path = "../packages/finschia-std" }
+schemars         = "0.8.12"
+serde            = { version = "1.0.167", default-features = false, features = ["derive"] }
+thiserror        = "1.0.43"
```

Comment:
Is it necessary to use Finschia package to instead.
https://github.com/Finschia/cw-workspace/blob/main/Cargo.toml

Reply:
This will be addressed in a separate PR.
https://github.com/Finschia/finschia-wasm/issues/6